includes/cognitive-services-azure-active-directory-authentication.md

```diff
@@ -9,15 +9,16 @@ ms.date: 07/23/2019
 ## Authenticate with Azure Active Directory
 
 > [!IMPORTANT]
-> Currently, **only** the Computer Vision API, Face API, Text Analytics API, and Immersive Reader support authentication using Azure Active Directory (AAD).
+> 1. Currently, **only** the Computer Vision API, Face API, Text Analytics API, and Immersive Reader support authentication using Azure Active Directory (AAD).
```

Comment:
the list of supported services includes: Computer Vision API, Face API, Text Analytics API, Immersive Reader, Form Recognizer, Anomaly Detector, and all Bing services except Bing Custom Search.

Reply:
@shuxianc - Please make the edits requested  by Greg, then we can merge your change.